src/EventNames.ts

```diff
@@ -0,0 +1,7 @@
+/**
+* Eventname class for use instead of hardcoded strings

```

Comment:
It's generally better to write descriptions of what a class *is* than to explain the alternatives.